app/helpers/users_helper.rb

```diff
@@ -113,7 +113,7 @@ def rtl_safe_username(user)
   def user_link(user, url_opts = {}, **link_opts)
     anchortext = link_opts[:anchortext]
     link_opts_reduced = { dir: 'ltr' }.merge(link_opts).except(:anchortext)
-    if user.nil? || (deleted_user?(user) && !moderator?)
+    if user.nil? || (deleted_user?(user) && !at_least_moderator?)
```

Comment:
Untested suggestion:

```suggestion
    if user.nil? || (deleted_user?(user) && standard?)
```

Reply:
As I mentioned in another comment, this one seems safe to me due to `user.nil? || ...` - let's see